packages/p2p-v2/src/components/FullPageMobileWrapper/FullPageMobileWrapper.scss

```diff
@@ -0,0 +1,37 @@
+.p2p-v2-mobile-wrapper {
```

Comment:
Full Page mobile wrapper for mobile screens:
<img width="1819" alt="Screenshot 2024-01-11 at 3 12 49 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/becc50d2-3b18-41cd-99af-2dfc129394ac">